src/iexamine.cpp

```diff
@@ -1420,6 +1420,38 @@ void iexamine::flower_bluebell(player &p, const tripoint &examp)
     return;
 }
 
+/**
+ * It's a flower, drink nectar if your able to.
+ */
+void iexamine::flower_tulip(player &p, const tripoint &examp)
```

Comment:
You need to put new flower function declarations to `iexamine.h`:

```CPP
void iexamine::flower_tulip(player &p, const tripoint &examp)
void iexamine::flower_spurge(player &p, const tripoint &examp)
```

Reply:
Thanks, I figured i missed somthing somewhere.